src/test/java/com/teragrep/pth10/TokenizerTest.java

```diff
@@ -142,4 +156,52 @@ public void tokenize3() {
                         }
                 );
     }
+
+    @Test
+    @DisabledIfSystemProperty(
+            named = "skipSparkTest",
+            matches = "true"
+    )
+    public void testRegexTokenizerNotSupportedFormat() {
+        Properties properties = new Properties();
+        properties.put("dpl.pth_06.bloom.pattern", "testRegex");
+        Config config = ConfigFactory.parseProperties(properties);
+
+        TokenizerStep step = new TokenizerStep(config, AbstractTokenizerStep.TokenizerFormat.STRING, "_raw", "result");
+        SparkSession spark = streamingTestUtil.getCtx().getSparkSession();
+        Assertions
+                .assertThrows(UnsupportedOperationException.class, () -> step.get(spark.read().schema(testSchema).csv(spark.emptyDataset(Encoders.STRING()))));
```

Comment:
Might be useful to assert the exception message as well

Reply:
added exception message assertion